### PR TITLE
feat(1692): api-catalog-spa: qa and user feedback changes

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Closes

1692

# Explain the feature/fix

- API create form owner field error shown and label changes
- Removed opc-footer
- Homepage ui improvement with new searchbar
- Typo fixes
- Updated readme

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

## Home Page

Screenshot 2021-11-26 at 2 31 53 PM

## Home Page on Search

Screenshot 2021-11-26 at 12 11 10 PM

## Owner field error state

Screenshot 2021-11-26 at 2 31 39 PM

## Empty env section

Screenshot 2021-11-26 at 3 05 11 PM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/api-catalog-spa/src/pages/ApiCUDPage/components/EnvironmentFormSection/EnvironmentFormSection.tsx [hunk 1]
```diff
@@ -98,6 +98,7 @@ export const EnvironmentFormSection = (): JSX.Element => {
                           placeholder="Enter env name"
                           isCreatable
                           onCreateOption={onEnvNameCreate}
+                          placeholderText="Enter env name"
```
Comment (riginoommen) [inline packages/api-catalog-spa/src/pages/ApiCUDPage/components/EnvironmentFormSection/EnvironmentFormSection.tsx:101]:
```suggestion
                          placeholderText="Enter environment name"
```

Comment (riginoommen) [reply packages/api-catalog-spa/src/pages/ApiCUDPage/components/EnvironmentFormSection/EnvironmentFormSection.tsx:101]: ENV in a tech shothand use full name to understand it

Comment (akhilmhdh) [reply packages/api-catalog-spa/src/pages/ApiCUDPage/components/EnvironmentFormSection/EnvironmentFormSection.tsx]: Done

**Review by riginoommen — APPROVED** [review]

**Comment by akhilmhdh** [comment]
Just updated 

`ApiCUDPage` owner select with better boolean expression and small error check issue

Ref: [Changed link](https://github.com/1-Platform/one-platform/pull/1316/files#diff-a04754a3b285b1f1101c3d18aba706c36b5d4fc59d3fd5d89b2e21e1b555f1aaR380)

**Review by ghanlohar — APPROVED** [review]
LGTM

**Review by riginoommen — APPROVED** [review]

**Pull request merged**